packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1397,6 +1398,14 @@ export interface CustomMessage extends MessageCommon {
 // @public
 export const darkTheme: PartialTheme & CallingTheme;
 
+// @public
+export type DeclarativeCallAgent = CallAgent & /* @conditional-compile-remove(one-to-n-calling) */ {
+    getIncomingCall: (incomingCallId: string) => DeclarativeIncomingCall;
```

Comment:
We can move this one to the top StatefulCallClient api, so DeclarativeCallAgent can just share the same set of api callAgent has - which is the principal of centralize all states and all additional apis in one place